src/entrypointConsumer/index.js

```diff
@@ -0,0 +1,112 @@
+class EntrypointConsumer {
+  constructor(connection, logger) {
+    this.connection = connection;
+    this.logger = logger;
+
+    /**
+     * Maps correlation ids to consumer tags
+     * @type {Object}
+     */
+    this.consumers = {};
+
+    /**
+     * Maps correlation ids to callbacks
+     * @type {Object}
+     */
+    this.callbacks = {};
+
+    this.consumeMessage = this.consumeMessage.bind(this);
+    this.addConsumer = this.addConsumer.bind(this);
+    this.removeConsumer = this.removeConsumer.bind(this);
+    this.removeAllConsumers = this.removeAllConsumers.bind(this);
+  }
+
+  /**
+   * Consume message from AMQ
```

Comment:
```suggestion
   * Consume message from RabbitMQ
```
We consume from RabbitmMQ